src/Core/src/Handlers/ScrollView/ScrollViewHandler.iOS.cs

```diff
@@ -5,30 +5,31 @@
 using CoreGraphics;
 using Microsoft.Maui.Graphics;
 using Microsoft.Maui.Layouts;
+using Microsoft.Maui.Platform;
 using ObjCRuntime;
 using UIKit;
 using Size = Microsoft.Maui.Graphics.Size;
 
 namespace Microsoft.Maui.Handlers
 {
-	public partial class ScrollViewHandler : ViewHandler<IScrollView, UIScrollView>
+	public partial class ScrollViewHandler : ViewHandler<IScrollView, MauiScrollView>
```

Comment:
Just leave this as `UIScrollView` 

Changing the generic type is a breaking API change so for now we aren't changing these types.

Reply:
Hmm but I needed to override the partial scrolling that UIScrollView does on the UITextFields and I did that inside the MauiScrollView. Is there a different way that I can accomplish this?

Reply:
Since this is for net8 maybe break it now? there shouldn't be that much people using this

Reply:
@tj-devel709 you don't have to change the generic types. You can just return the `MauiScrollView` from `CreatePlatformView` and all will be fine.

Reply:
> Since this is for net8 maybe break it now? there shouldn't be that much people using this

AFAIK we have pretty strict rules around ABI breaks like this. So, I don't think we really can just break the binaries. We need to figure out an official path here with deprecating handlers as we move through .NET releases.

Reply:
Reverted these changes and now return the MauiScrollView from the CreatePlatformView as Shane said